.vscode/settings.json

```diff
@@ -1,3 +1,4 @@
 {
-    "go.useLanguageServer": true
+    "go.useLanguageServer": true,
```

Comment:
this flie probable should be removed and setting shoudl fo into gitpod.theia-workspace file. I added it by mistake in some PR.

Reply:
Hmm, I see, thanks for letting me know!